SampleGallery/Samples/SDK Insider/Gears/Gears.xaml

```diff
@@ -1,46 +1,24 @@
-            <Image x:Name="Gear17" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="691" Canvas.Top="173"/>
-            <Image x:Name="Gear18" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="713" Canvas.Top="260"/>
-            <Image x:Name="Gear19" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="790" Canvas.Top="306"/>
-            <Image x:Name="Gear20" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="837" Canvas.Top="229"/>
-            <Image x:Name="Gear21" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="862" Canvas.Top="142"/>
-            <Image x:Name="Gear22" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="887" Canvas.Top="55"/>
-            <Image x:Name="Gear23" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="963" Canvas.Top="101"/>
-            <Image x:Name="Gear24" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="1050" Canvas.Top="79"/>
-            <Image x:Name="Gear25" Source="ms-appx:///Assets/gear.png" HorizontalAlignment="Left" VerticalAlignment="Top" Width="100" Height="100" Canvas.Left="1072" Canvas.Top="165"/>
-            </Canvas>
-        </Grid>
-</local:SamplePage>
+            <Button x:Name="AnimateSlow" Click="AnimateSlow_Click" Background="White" Width="100" Height="50" VerticalAlignment="Top" Margin="10,10,0,0">Slow</Button>
+            <Button x:Name="AnimateFast" Click="AnimateFast_Click" Background="White" Width="100" Height="50" VerticalAlignment="Top" Margin="10,10,0,0">Fast</Button>
+            <Button x:Name="Stop" Click="Stop_Click" Background="White" Width="100" Height="50" VerticalAlignment="Top" Margin="10,10,0,0">Stop</Button>
+            <Button x:Name="Reverse" Click="Reverse_Click" Background="White" Width="100" Height="50" VerticalAlignment="Top" Margin="10,10,0,0">Reverse</Button>
+            <Button x:Name="AddGearButton" Click="AddGear_Click" Background="White" Width="100" Height="50" VerticalAlignment="Top" Margin="10,10,0,0">Add Gear</Button>
+            <TextBox x:Name="NumberOfGears" Width="200" Height="50" Margin="10" VerticalAlignment="Top" />
+            <Button x:Name="AddXGearsButton" Width="100" Height="50" Margin="10" VerticalAlignment="Top" Background="White" Content="Add X Gears" Click="AddXGearsButton_Click" />
+            <TextBlock x:Name="GearCount" Text="{x:Bind Count, Mode=OneWay}" FontSize="32" Foreground="White" />
+        </StackPanel>
```

Comment:
Cleanup empty lines (i.e. before </StackPanel>, and above/below <Image x:Name="FirstGear".../>